frontend/ui/data/css_tweaks.lua

```diff
@@ -478,6 +478,24 @@ sup { font-size: 50% !important; vertical-align: super !important; }
 sub { font-size: 50% !important; vertical-align: sub !important; }
                 ]],
             },
+            (function()
+                local sub_table = {
+                    title = _("Override normal font line leight"),
```

Comment:
Might need a better label, because if I hadn't read the PR, this would just look like it's simply enforcing the actual standard line-height CSS property ;).